src/main/java/uk/gov/hmcts/reform/draftstore/service/secrets/Secrets.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.draftstore.service.secrets;
+
+import com.google.common.base.Strings;
+import uk.gov.hmcts.reform.draftstore.utils.Lists;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.Arrays.asList;
+import static java.util.stream.Collectors.toList;
+
+public class Secrets {
+
+    public static final int MIN_SECRET_LENGTH = 16;
+    public static final String SEPARATOR = ",";
+
+    public final String primary;
+    public final String secondary;
+
+    public Secrets(String primary, String secondary) {
```

Comment:
Maybe secondary should be wrapped in Optional, so that it would be obvious this one is not required.

Reply:
I take that back, seeing that both are optional.